packages/history/README.md

```diff
@@ -0,0 +1,3 @@
+# history
+
+TODO
```

Comment:
Something to keep in mind with this file is it'll only ever be read in a few contexts:

- on npmjs.org (which won't happen now since we aren't publishing this to npm at this time)
- after someone `npm install`s this package (again, not gonna happen w/out publishing to npm)
- on GitHub (most likely)

So for now it can be pretty basic and just point people back to the repo.